src/lib/caching/structures/guilds/GuildMember.ts

```diff
@@ -94,6 +94,24 @@ export class GuildMember extends Structure {
 		return this.nick ?? this.user?.username ?? null;
 	}
 
+	/**
+	 * The displayed colour of the member.
+	 * @since 0.0.4
+	 */
+	public get displayColour(): number {
```

Comment:
```suggestion
	public get displayColor(): number {
```

Reply:
False

Reply:
This is actually more consistent with the rest of the lib. For example, `highestRole.color` as opposed to `highestRole.colour`